core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -37,6 +35,22 @@ class HttpSamDAO(baseSamServiceURL: String)(implicit val system: ActorSystem) ex
     }
   }
 
+  override def userHasAction(resourceTypeName: SamResourceTypeName, resourceId: String, action: SamResourceAction, userInfo: UserInfo): Future[Boolean] = {
+    val url = samServiceURL + s"/api/resource/${resourceTypeName.value}/${resourceId}/action/${action.value}"
+    val httpRequest = Get(url)
+    val pipeline = addAuthHeader(userInfo) ~> sendReceive
+    val result: Future[HttpResponse] = pipeline(httpRequest)
+
+    retry(when500) { () =>
+      result.map { response =>
+        response.status match {
+          case s if s.isSuccess => response.entity.asString.toBoolean
```

Comment:
i tried every possible way (except for the way that will actually work) to get this to unmarshal as a boolean. this line is a hack so i could move forward and field solutions